templates/components/products/price.html

```diff
@@ -44,10 +68,12 @@
     {{/if}}
     {{#if page_type '===' 'product'}}
         {{#if price.saved}}
-            <div class="price-section price-section--saving">
-                <span class="price">
-                    {{lang 'products.you_save' amount=price.saved.formatted}}
+            <div class="price-section price-section--saving price">
+                <span class="price">{{lang 'products.you_save_1'}}</span>
+                <span data-product-price-saved class="price">
+                    {{price.saved.formatted}}
                 </span>
+                <span class="price">{{lang 'products.you_save_2'}}</span>
```

Comment:
Please update this based on the suggestions above

Reply:
That unfortunately does not work, and is causing the bug CATALOG-3121. Which is the main reason for the split

Reply:
yeah, this seems fine because we are dealing with updating the already-rendered view here and so can't pass it to the helper at this point.  but let's name these something more descriptive

Reply:
i'm wondering - maybe we could simply have something like a you_save = "You save" and then include the parens as part of the html or css so we could have:

```
<span class="price">{{lang 'products.you_save'}}
<span data-product-price-saved class="price">
  {{price.saved.formatted}}
</span>
```

::before {
  content: "(";
}

::after {
  content: ")";
}

Reply:
Easiest would be in html just in case css messes up. You would always be 100% sure it works with html

Reply:
splitting it in en.json seems the best for now. But we will need to fix the underlining problem of having a dynamic value to be added as a parameter.